docs/faq/onboarding/extension.md

```diff
@@ -40,17 +40,59 @@ The Universal Profile Browser Extension currently does not feature a way to impo
 
 :::
 
-## How to check if 2FA is set up correctly?
+## Can I add 2FA recovery to an existing Universal Profile?
 
-You can check if the recovery is set up correctly using the Universal Profile Browser Extension. If you visit the Controller Menu, there need to be at least 3 Controllers:
+At the current state (11th January 2024), the 2FA recovery method from LUKSO can only be added during the creation of a profile on [UniversalProfile.cloud](https://universalprofile.cloud/).
+
+## How to check if 2FA recovery is set up correctly?
+
+If you ever want to recover your profile, a fresh controller key will be added to this profile from a new extension setup. You must verify multiple steps to check if this new controller key can be added to your Universal Profile by the recovery service:
```

Comment:
Not sure about using the word "key" alongside the word "controller". Would remove it for consistency with next paragraph

```suggestion
If you ever want to recover your profile, a new controller will be added to this profile from a new extension setup. You must verify multiple steps to check if this new controller can be added to your Universal Profile by the recovery service:
```